### DISCUSSION TITLE
how to trigger onscroll on window from overlay scrollbar

### DISCUSSION THREAD

**Discussion by reachruc** [KingSora/OverlayScrollbars#454] [Q&A]
hi 
i am trying to use overlay scrollbar in my react app, but after many tries i am not able to trigger scroll event on the window from overlay scrollbar. could you direct me towards the right documentation or solution? thanks

**Comment by KingSora** [comment]
@reachruc In `v1` this is a bit tricky to achieve, because of the limitations of that version.

`v2` just released and there it just works without any problems. :)
